src/lib.rs

```diff
@@ -5,44 +5,65 @@ use std::sync::mpsc;
 use std::{fs, process};
 
 use anyhow::{bail, ensure, Context, Result};
+use lazy_static::lazy_static;
 use notify::{RecommendedWatcher, RecursiveMode, Watcher};
 use structopt::StructOpt;
 use wasm_bindgen_cli_support::Bindgen;
 
+pub fn metadata() -> &'static cargo_metadata::Metadata {
+    lazy_static! {
+        static ref METADATA: cargo_metadata::Metadata = cargo_metadata::MetadataCommand::new()
+            .exec()
+            .expect("cannot get crate's metadata");
+    }
+
+    &METADATA
+}
+
+pub fn package(name: &str) -> Option<&cargo_metadata::Package> {
+    metadata().packages.iter().find(|x| x.name == name)
+}
+
 #[derive(Debug, StructOpt)]
 pub struct Build {
     #[structopt(long)]
-    release: bool,
+    pub release: bool,
+    #[structopt(skip = default_build_command())]
```

Comment:
Put some vertical space to separate structopt arguments from others
```suggestion
    pub release: bool,

    #[structopt(skip = default_build_command())]
```